.github/workflows/code_quality.yml

```diff
@@ -11,7 +11,6 @@ on:
     paths-ignore:
       - '**/release_draft.yml'
       - '**/maven*.yml'
-      - '**/*.md'
```

Comment:
In addition to all proposed changes in this PR, can we add an additional step to affected workflows where we check for doc files and successfully completes task if needed?

https://github.com/marketplace/actions/changed-files